src/components/formik/FormikContainer.jsx

```diff
@@ -89,24 +96,25 @@ export const FormikContainer = (props) => {
           const { resetForm } = formikProps;
           return (
             <>
-              <FormHeader formTitle={formTitle}>
-                {dataHook(id).data.__licenseeConnection__ === undefined ||
-                dataHook(id).data.__licenseeConnection__.length === 0 ? (
-                  <WarningButton
-                    id="delete"
-                    onClick={() => setConfirmationDialogOpen(true)}
-                  >
-                    Delete
-                  </WarningButton>
-                ) : (
-                  <DisabledButton id="delete-disabled">Delete</DisabledButton>
-                )}
-                {editMode ? null : (
+              <Box sx={{ display: hidden() }}>

```

Comment:
This location of Box currently removes the form title. Moving it inside FormHeader.